src/app/features/task-list/task-list.component.ts

```diff
@@ -34,25 +32,20 @@ export class TaskListComponent implements OnInit, OnDestroy {
         this.taskService.taskSubject.next(tasks);
       });
 
-    this.tasks$ = this.route.paramMap.pipe(
+    this.route.paramMap.pipe(
       switchMap(params => {
         this.taskId = Number(params.get('id'));
         return this.taskService.tasks$;
```

Comment:
you're essentially returning tasks$ to nothing, why did you delete the tasks$ observable from before?